mongorestore/local/test_auto_idx.metadata.json

```diff
@@ -0,0 +1 @@
+{"options":{"autoIndexId":false},"indexes":[],"uuid":"0715c8e614a747d8baf5853ab381a303"}
```

Comment:
I think you might want this and its corresponding bson file to be in `mongorestore/testdata/local/` instead of `mongorestore/local/`